projects/angular/src/utils/popover/providers/popover-events.service.ts

```diff
@@ -42,16 +43,18 @@ export class ClrPopoverEventsService implements OnDestroy {
     );
   }
 
-  private scrollSubscription: Subscription;
+  private scrollSubscription: Subscription | null = null;
   public addScrollListener() {
     if (this.scrollToClose) {
-      this.documentScroller = fromEvent(this.document, 'scroll', { capture: true });
-      this.scrollSubscription = this.documentScroller
-        .pipe(filter(this.testForSmartPopoverContentContainer))
-        .subscribe(() => {
-          this.smartOpenService.open = false;
+      const documentScroller$ = fromEvent(this.document, 'scroll', { capture: true, passive: true });
```

Comment:
Why do we set `passive` on the `scroll` event itself? 
If I understood it correctly, it's usually used with other events that interfere with scrolling (like `touchstart`, `click`..) to not wait for their completion to start scrolling. I don't get the idea of using in on `scroll`.

Reply:
Initially, this was intended to be used with `scroll` and `touch*` events. The purpose of passive event listeners is to unblock the composite thread (while the handler which is being executed doesn't block it).

This is forcerly set to `true` in latest Chromium versions only for the `document`:
```cpp
bool EventTarget::IsTopLevelNode() {
  ...
  if (node->IsDocumentNode())
    return true;
  }
}

if (IsWheelScrollBlockingEvent(event_type) && IsTopLevelNode()) {
  if (!options->hasPassive()) {
    options->setPassive(true);
  }
}
```
(See that it's checking if the `node` is `document` and it's a scroll event).

This is not set to `true` by default in Servo (Mozilla engine) and WebKit.